name.abuchen.portfolio/src/name/abuchen/portfolio/datatransfer/pdf/TradeRepublicPDFExtractor.java

```diff
@@ -1575,7 +1586,7 @@ private void addAccountStatementTransaction_Format02()
 
                         .optionalOneOf( //
                                         // @formatter:off
-                                        // 28 
```

Comment:
die Freizeichen vor dem Zeilenwechsel sind real vorhanden und in den RegExPattern enthalten. Hab ich bewusst da gelassen.

Reply:
Ahh.. das macht bei mir das Bereinigen des Quelltextes automatisch weg... ist aber nicht schlimm.